generators/app/templates/_steps.js

```diff
@@ -10,16 +10,33 @@
   express = require('express');
   require('../../spec/factories/index');
 
+  function clearRoutes(_this) {
```

Comment:
used steps from the new `crowdtap.missions` app. It uses the new retry logic when checking for api resqusets